beacon_node/beacon_chain/src/attestation_rewards.rs

```diff
@@ -47,8 +47,10 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             .state_root_at_slot(state_slot)?
             .ok_or(BeaconChainError::NoStateForSlot(state_slot))?;
 
+        // This branch is reached from the HTTP API. We assume the user wants
```

Comment:
Can you add some tag to all these comments? Like
```suggestion
        // `update_cache = true` This branch is reached from the HTTP API. We assume the user wants
```
or
```suggestion
        // `get_state(.., true)` This branch is reached from the HTTP API. We assume the user wants
```
Will help future generations

Reply:
I don't think is really helpful compared to just grepping for `get_state`. I feel like we're bikeshedding these comments a bit at this point